js/account.js

```diff
@@ -6,6 +6,20 @@ export const AccountAPI = superclass =>
             return contents;
         }
 
+        async updateSelfAccount(account) {
+            const url = this.baseUrl + "accounts/me";
+
+            const dataM = {};
+            if (account.avatar) dataM.avatar = account.avatar;
+            if (account.description) dataM.description = account.description;
+            if (account.meta_extra) dataM.meta_extra = JSON.stringify(account.meta_extra);
+
+            const contents = await this.put(url, {
+                dataM: dataM

```

Comment:
Is there any problem with doing `dataM: account`?

Reply:
I know it's different from ripe-sdk, but at the same time it's coherent with the rest of these yonius API clients.

Reply:
I wanted to make it similar to ripe-sdk, and also the JSON.stringify part. Do I pass the account and override the meta_extra?